dev/vcpkg/ports/libelf/portfile.cmake

```diff
@@ -14,10 +14,22 @@ vcpkg_extract_source_archive(
 file(DOWNLOAD "https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD"
      "${SOURCE_PATH}/config.guess"
      STATUS status_guess)
+list(GET status_guess 0 status_code)
+list(GET status_guess 1 status_string)
+if(NOT status_code EQUAL 0)
```

Comment:
we may should skip the download, if local file already exists? 
The other issues on libelf seems due to downloading some empty config.guess/config.sub

Reply:
> we may should skip the download, if local file already exists?

+1

Reply:
If the download fails, an empty file will be generated.